### PR TITLE
List deployment with pagination

### PR DESCRIPTION
**Pull request by kooyunmo**
This PR applies the pagination API when fetching the deployment list.

### CODE REVIEW

File: pfcli/deployment.py [hunk 1]
```diff
@@ -275,15 +275,21 @@ def list(
...
+    num_active_deployments = len(deployments)
+    if include_terminated and limit > num_active_deployments:
         deployments += client.list_deployments(
-            str(project_id) if not org else None, True
-        )["deployments"]
+            project_id=str(project_id) if not org else None,
+            archived=True,
+            limit=limit - num_active_deployments,
+            from_oldest=from_oldest,
+        )
```
Comment (kdh0102) [inline pfcli/deployment.py:291]: I am OK with the current order, but eventually `archived=None` field should be available and deployments including both active and archived should be ordered by timestamp.
Currently, it seems to fill the remaining spaces with archived deployments.

Also, it looks good to let users know that the output order prioritizes active deployments. Maybe, we can refine `include_terminated` option description.

**Review by kdh0102 — APPROVED** [review]
LGTM! Please check my minor comment before you merge it.

**Review by kdh0102 — APPROVED** [review]
LGTM! Please check my minor comment before you merge it.

File: pfcli/deployment.py (hunk 1 above)
Comment (kooyunmo) [reply pfcli/deployment.py:291]: I agree with your suggestion. I thought that there is no `archived=None` option 😅. I will change it as you said.

Comment (kooyunmo) [reply pfcli/deployment.py:291]: Oh, I see. Then I will merge this PR as is.

**Pull request merged**